.travis.yml

```diff
@@ -16,25 +16,33 @@ addons:
     - python-numpy
     - python-cython
     - python-dev
-
+  artifacts:
+    debug: true
+    paths:
+    - $HOME/tests/test_data/*_results
+  s3_region: us-west-1
 install:
-  # Hack so python bindings can find locally installed library file
-  - export LD_LIBRARY_PATH=$TRAVIS_BUILD_DIR/libswiftnav/build/install/usr/local/lib
-  - git submodule update --init
-
+- export LD_LIBRARY_PATH=$TRAVIS_BUILD_DIR/libswiftnav/build/install/usr/local/lib
+- git submodule update --init
 script:
-  # Build libswiftnav
-  - cd libswiftnav/
-  - mkdir build
-  - cd build/
-  - cmake ../
-  - make
-  # Install locally to avoid sudo
-  - make DESTDIR="./install" install
-  - cd ../python
-  - pip install cython
-  - python setup.py build && python setup.py install
-  - cd $TRAVIS_BUILD_DIR
-  - pip install -r requirements.txt
-  - python setup.py develop
-  - py.test -v tests/
+- cd libswiftnav/
```

Comment:
Oh yeah, the travis encryption tool reformatted this when I added the encrypted variables, I'll fix it.